### PR TITLE
To verify kdump on RHEL and Ubuntu -v2

### PR DESCRIPTION
**Pull request by Pavithra1602**
This test triggers kdump on the machine configured in config.yaml and checks if vmcore is generated.

Signed-off-by: Pavithra <pavrampu@linux.vnet.ibm.com>

### CODE REVIEW

File: generic/kdump.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+class KDUMP(Test):
+
+    def run_cmd_out(self, cmd):
+        return process.system_output(cmd, shell=True, ignore_status=True, sudo=True).strip()
+
+    def setUp(self):
+        sm = SoftwareManager()
+        if not sm.check_installed("openssh*") and not sm.install("openssh*"):
+            self.error("Fail to install openssh required for this test.")
+        self.ip = self.params.get('ip', default='')
```
Comment (PraveenPenguin) [inline generic/kdump.py]: can we check if IP is not valid skip the test ?

File: generic/kdump.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        self.user_name = self.params.get('user_name')
+        self.password = self.params.get('password')
+        self.prompt = self.params.get('prompt', default='')
+
+    def test(self):
+        log_file = os.path.join(self.srcdir, "file")
+        session0 = remote.RemoteRunner("ssh", self.ip, 22, self.user_name, self.password,
```
Comment (PraveenPenguin) [inline generic/kdump.py]: can you please use valid name like session_target ?

File: generic/kdump.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            session4 = remote.RemoteRunner("ssh", self.ip, 22, self.user_name, self.password,
+                                           self.prompt, "\n", log_file, 100, 10, None)
+            session4.run("kdump-config show", 600, "True")
+            if self.run_cmd_out("cat %s | grep -Eai 'Not ready to'" % log_file):
+                self.fail("Kdump is not operational")
+            else:
+                self.log.info("Kdump status is operational")
+        if "rhel" in open(log_file).read():
+            file_list = ['vmcore-dmesg.txt', 'vmcore']
+            f = 11
```
Comment (PraveenPenguin) [inline generic/kdump.py]: please use valid variable name atleat 3 char

**Comment by Pavithra1602** [comment]
raised new PR

**Pull request closed without merging**